sea-orm-migration/src/cli.rs

```diff
@@ -53,35 +53,41 @@ where
             .init()
     };
     match matches.subcommand() {
-        ("fresh", _) => M::fresh(db).await,
-        ("refresh", _) => M::refresh(db).await,
-        ("reset", _) => M::reset(db).await,
-        ("status", _) => M::status(db).await,
-        ("up", None) => M::up(db, None).await,
-        ("down", None) => M::down(db, Some(1)).await,
-        ("up", Some(args)) => {
-            let str = args.value_of("NUM_MIGRATION").unwrap_or_default();
-            let steps = str.parse().ok();
-            M::up(db, steps).await
-        }
-        ("down", Some(args)) => {
-            let str = args.value_of("NUM_MIGRATION").unwrap();
-            let steps = str.parse().ok().unwrap_or(1);
-            M::down(db, Some(steps)).await
-        }
-        _ => M::up(db, None).await,
+        Some((cmd, args)) => match cmd {
```

Comment:
Might be rewriting this with clap derive feature?

Reply:
Thank you!

Reply:
Thanks! @smonv

Reply:
derive code added for `sea-orm-migration`

Reply:
well, about `sea-orm-cli` tests, I'm still not sure how to done it correctly, maybe some one have insight about it?